WordPressAuthenticator/Signin/Login2FAViewController.swift

```diff
@@ -178,6 +178,8 @@ class Login2FAViewController: LoginViewController, NUXKeyboardResponder, UITextF
 
         // Disconnect now that we're done with Google.
         GIDSignIn.sharedInstance().disconnect()
+
+        WordPressAuthenticator.track(.signedIn)
```

Comment:
I think we can add a `["source": "google"]` here since this is only used by Google as far as I can tell. Thoughts?

Reply:
I'm happy to do that if that's the case, I just couldn't work out if it was used by not-Google logins. There are no other references to Google in here other than line 180, and the `loginSocialSuccess` call below doesn't mention Google so I wasn't sure!

Reply:
Well, I just tried with an Apple account and was finally able to repro https://github.com/wordpress-mobile/WordPress-iOS/issues/12477! 🤦‍♂ 

I'd be ok with leaving it generic for now. Currently, it is only google, but hard-coding that is probably dangerous and might be wrong when 12477 is fixed.

Reply:
👍